### DISCUSSION TITLE
Rule Trigger Analog Port Question

### DISCUSSION THREAD

**Discussion by tsunfinger** [arendst/Tasmota#19622] [Q&A]
Short question to documentation:
**_Analog#A0div10 | when the A0 input changes by more than 1% it provides a value between 0 and 100_**
Where 1% comes from? Is _**div**number_  in thousandth or is 1% fixed?
When value of A0 will be used/set for reference in percentage changes? 

Thanks
Tom

**Comment by sfromis** [comment]
The `div` number is fixed and came from dividing the ESP8266 analog pin value 0..1023 by 10 to get to a percentage, capped at 100.

When I say "fixed", this does not always mean 10, as ESP32 has more bits, and still an approximation of 0..100% when using `div10`.

**Reply by tsunfinger** [reply]
_18:59:54.982 MQT: tele/sonoff/SENSOR = {"Time":"2023-09-29T18:59:54","ANALOG":{"A1":1618,"A2":4095}}_
Ok, my values are between 0-4095. 

Thanks for clarification
Tom
